#goal definition
string target_group
geometry_msgs/PointStamped target_point
PlanConfig config

---
#result definition

ComputePathResult plan_result
---
#feedback definition
